docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -113,7 +98,7 @@ $ xtrabackup --prepare --target-dir=/data/backup \
 --keyring-vault-config=/etc/vault.cnf
 ```
 
-Review [using the keyring vault plugin](https://www.percona.com/doc/percona-server/LATEST/security/using-keyring-plugin.html#using-keyring-plugin) for a description of keyring vault plugin settings.
+Review [Use the keyring component or keyring plugin](https://docs.percona.com/percona-server/8.0/using-keyring-plugin.html) for a description of keyring vault plugin settings.
```

Comment:
I think we removed the plugin - check

Reply:
In https://jira.percona.com/browse/PS-8854 they say that `keyring_vault plugin should be converted to a component`

Reply:
I left a comment regarding removing keyring_vault plugin